src/error.rs

```diff
@@ -1,22 +1,43 @@
 use thiserror::Error;
 
 /// Error type for this crate.
-#[derive(Debug, Error, PartialEq)]
+#[derive(Debug, Error)]
```

Comment:
Why did you remove the `PartialEq` derivation ? Does this prevent from mapping on different cases ?

Reply:
Because it was only used in the tests and I think some errors from `leb128` doesn't implement it.